dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10JaxWithLogging/server.xml

```diff
@@ -0,0 +1,21 @@
+<server description="Server for testing Telemetry10">
+
+    <include location="../fatTestPorts.xml" />
+
+    <featureManager>
+        <feature>servlet-6.0</feature>
+        <feature>mpTelemetry-1.0</feature>
+        <feature>componentTest-2.0</feature>
+        <feature>restfulWS-3.1</feature>
+        <feature>mpRestClient-3.0</feature>
+    </featureManager>
+
+    <javaPermission className="java.lang.RuntimePermission" name="getClassLoader"/>
```

Comment:
We think this was needed for a call from feature code.

If that's the case, we should have a doPriv there and this permission should be removed.

We need this test to pass without this permission (unless we find that test code needs it).

Reply:
This permission line needs to be removed (or justified) before this PR can be merged.